kolibri/core/assets/src/views/StorageNotification.vue

```diff
@@ -121,7 +123,11 @@
         }
       },
       manageChannel() {
-        this.$router.push('/');
+        if (!this.isLearner) {
```

Comment:
Hi @AllanOXDi - there are a couple more things here that I talked about with @rtibbles.

1. Using `canManageContent` is a more specific permission than checking that the user is not a learner, so you can use this (an import the getter that goes along with it)
2. Adding a check to make sure the URL exists would help with some of the routing concerns that Richard mentioned previously

```
 if (this.canManageContent && urls['kolibri:kolibri.plugins.device:device_management'])
```

Let us know if you have further questions

Reply:
Hi @marcellamaki  I think I have already  implemented  `canManageContent` in this[ PR ](https://github.com/learningequality/kolibri/pull/10709/files#:~:text=...mapGetters(%5B%27isLearner%27%2C%20%27isAdmin%27%2C%20%27canManageContent%27%5D)%2C)for permission checking.  I am assuming that this PR is following it?

Reply:
I will first wait for the PR to get merge and get to this.